### PR TITLE
Add operator cross-referencing to video processing tutorials

### PR DESCRIPTION
**Pull request by JanuszL**
Signed-off-by: Janusz Lisiecki <jlisiecki@nvidia.com>

## Category:

**Other** (*e.g. Documentation, Tests, Configuration*)

## Description:
- adds operator cross-referencing to video processing tutorials

### Affected modules and functionalities:
- examples/sequence_processing/index.py

### Key points relevant for the review:
- NA

### Tests
- [ ] Existing tests apply
- [ ] New tests added
  - [ ] Python tests
  - [ ] GTests
  - [ ] Benchmark
  - [ ] Other
- [x] N/A

### Documentation
- [ ] Existing documentation applies
- [x] Documentation updated
  - [ ] Docstring
  - [ ] Doxygen
  - [x] RST
  - [ ] Jupyter
  - [ ] Other
- [ ] N/A

#### Requirements
- [ ] Implements new requirements
- [ ] Affects existing requirements
- [x] N/A

**REQ IDs**: N/A

**JIRA TASK**: DALI-2720

### CODE REVIEW

**Comment by JanuszL** [comment]
!build

File: docs/examples/sequence_processing/index.py [hunk 1]
```diff
@@ -1,9 +1,16 @@
...
+                  op_reference('fn.readers.video', "Tutorial describing how to use video reader")),
+        doc_entry("video/video_reader_label_example.ipynb",
+                  op_reference('fn.readers.video', "Tutorial describing how to use video reader to output frames with labels")),
+        doc_entry("video/video_file_list_outputs.ipynb",
+                  op_reference('fn.readers.video', "Tutorial describing how output to frames with \
+                                lables assigned to dedicated ranges of frame numbers/timestamps")),
+        doc_entry("sequence_reader_simple_example.ipynb",
+                  op_reference('fn.readers.sequence', "Tutorial describing how to read sequence of video frames stores as separete files")),
+        doc_entry("optical_flow_example.ipynb",
+                  [op_reference('fn.readers.video', "Tutorial describing how read video and calculate optical flow"),
```
Comment (jantonguirao) [inline docs/examples/sequence_processing/index.py:14]:
```suggestion
                  [op_reference('fn.readers.video', "Tutorial describing how to calculate optical flow from video inputs"),
```

File: docs/examples/sequence_processing/index.py [hunk 2]
```diff
@@ -1,9 +1,16 @@
...
+                   op_reference('fn.optical_flow', "Tutorial describing how to calculate optical flow from a sequence of frames")]),
```
Comment (jantonguirao) [inline docs/examples/sequence_processing/index.py:15]:
```suggestion
                   op_reference('fn.optical_flow', "Tutorial describing how to calculate optical flow from sequence inputs")]),
```

File: docs/examples/sequence_processing/index.py [hunk 3]
```diff
@@ -1,9 +1,16 @@
...
-        "optical_flow_example.ipynb",
+        doc_entry("video/video_reader_simple_example.ipynb",
+                  op_reference('fn.readers.video', "Tutorial describing how to use video reader")),
+        doc_entry("video/video_reader_label_example.ipynb",
+                  op_reference('fn.readers.video', "Tutorial describing how to use video reader to output frames with labels")),
+        doc_entry("video/video_file_list_outputs.ipynb",
+                  op_reference('fn.readers.video', "Tutorial describing how output to frames with \
+                                lables assigned to dedicated ranges of frame numbers/timestamps")),
+        doc_entry("sequence_reader_simple_example.ipynb",
+                  op_reference('fn.readers.sequence', "Tutorial describing how to read sequence of video frames stores as separete files")),
```
Comment (jantonguirao) [inline docs/examples/sequence_processing/index.py:12]:
```suggestion
                  op_reference('fn.readers.sequence', "Tutorial describing how to read sequence of video frames stored as separate files")),
```

File: docs/examples/sequence_processing/index.py [hunk 4]
```diff
@@ -1,9 +1,16 @@
...
-        "video/video_file_list_outputs.ipynb",
-        "sequence_reader_simple_example.ipynb",
-        "optical_flow_example.ipynb",
+        doc_entry("video/video_reader_simple_example.ipynb",
+                  op_reference('fn.readers.video', "Tutorial describing how to use video reader")),
+        doc_entry("video/video_reader_label_example.ipynb",
+                  op_reference('fn.readers.video', "Tutorial describing how to use video reader to output frames with labels")),
+        doc_entry("video/video_file_list_outputs.ipynb",
+                  op_reference('fn.readers.video', "Tutorial describing how output to frames with \
+                                lables assigned to dedicated ranges of frame numbers/timestamps")),
```
Comment (jantonguirao) [inline docs/examples/sequence_processing/index.py:10]:
```suggestion
                  op_reference('fn.readers.video', "Tutorial describing how to output frames with \
                                labels assigned to dedicated ranges of frame numbers/timestamps")),
```

Comment (JanuszL) [reply docs/examples/sequence_processing/index.py]: Done

File: docs/examples/sequence_processing/index.py (hunk 1 above)
Comment (JanuszL) [reply docs/examples/sequence_processing/index.py]: Done

**Comment by JanuszL** [comment]
!build

**Comment by JanuszL** [comment]
!build

**Review by jantonguirao — APPROVED** [review]

**Review by stiepan — APPROVED** [review]

**Pull request merged**